examples/dynamic_rendering/source/dynamic_rendering.cpp

```diff
@@ -0,0 +1,244 @@
+#include "avk/attachment.hpp"
+#include "avk/command_buffer.hpp"
+#include "avk/commands.hpp"
+#include "avk/layout.hpp"
+#include "avk/memory_access.hpp"
+#include "avk/pipeline_stage.hpp"
+#include "imgui.h"
+
+#include "configure_and_compose.hpp"
+#include "imgui_manager.hpp"
+#include "invokee.hpp"
+#include "sequential_invoker.hpp"
+
+#include "context_vulkan.hpp"
+#include <string>
+#include <vulkan/vulkan_core.h>
+#include <vulkan/vulkan_structs.hpp>
+
+class dynamic_rendering_app : public avk::invokee
+{
+public: // v== xk::invokee overrides which will be invoked by the framework ==v
```

Comment:
```suggestion
public: // v== avk::invokee overrides which will be invoked by the framework ==v
```

This is an artefact of a veeery long time ago 🙈